app/routes/user/permissions.tsx

```diff
@@ -0,0 +1,300 @@
+/*!
+ * Copyright © 2022 United States Government as represented by the Administrator
+ * of the National Aeronautics and Space Administration. No copyright is claimed
+ * in the United States under Title 17, U.S. Code. All Other Rights Reserved.
+ *
+ * SPDX-License-Identifier: NASA-1.3
+ */
+
+import type { DataFunctionArgs } from '@remix-run/node'
+import { Form, useFetcher, useLoaderData } from '@remix-run/react'
+import {
+  Button,
+  ButtonGroup,
+  ComboBox,
+  Grid,
+  Label,
+} from '@trussworks/react-uswds'
+import { useState } from 'react'
+import SegmentedCards from '~/components/SegmentedCards'
+import { getFormDataString } from '~/lib/utils'
+import { getUser } from '../__auth/user.server'
+import type { EndorsementRequest } from './endorsements.server'
+import { EndorsementsServer } from './endorsements.server'
+
+type intermediateMappedUser = {
```

Comment:
What does "intermediate" mean here?

```suggestion
type IntermediateMappedUser = {
```